### ISSUE TITLE
Screenshots from a test student TP

### ISSUE THREAD

**Issue by bohungi** [rw-moore/SciLO#123]
This is what I see after logging into the quiz system:

![image](https://user-images.githubusercontent.com/9791639/157097387-ad15b02b-6438-4d46-a930-a62627745f26.png)

It hangs here indefinitely.

![image](https://user-images.githubusercontent.com/9791639/157097517-884dd2b6-38cc-4c2b-8a06-c06373548e76.png)

**Comment by corlick98** [comment]
I had found this issue on Friday and I thought I had sent the fix to the live system, but it appears I did not. It should be working now.

**Comment by bohungi** [comment]
Indeed, now the quiz appears ... but: Students will be extremely grateful for this presentation of a quiz: Solution is presented along with the question. 
Perhaps this will automatically fix itself once the interface for feedback configuration is in place?
